backend/infrahub/testing/helpers.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+
+import pytest
+from infrahub_sdk import Config, InfrahubClient, InfrahubClientSync
+from prefect.client.orchestration import PrefectClient
+
+from .container import InfrahubDockerCompose
+
+
+class TestInfrahub:
+    @pytest.fixture(scope="class")
+    def tmp_directory(self, tmpdir_factory: pytest.TempdirFactory) -> Path:
+        directory = Path(str(tmpdir_factory.getbasetemp().strpath))
+        return directory
+
+    @pytest.fixture(scope="class")
+    def default_branch(self) -> str:
+        return "main"
+
+    @pytest.fixture(scope="class")
+    def infrahub_compose(self, tmp_directory: Path) -> InfrahubDockerCompose:
+        return InfrahubDockerCompose.init(directory=tmp_directory)
+
+    @pytest.fixture(scope="class")
+    def infrahub_app(self, request: pytest.FixtureRequest, infrahub_compose: InfrahubDockerCompose) -> dict[str, int]:
+        def cleanup() -> None:
+            infrahub_compose.stop()
+
+        infrahub_compose.start()
+        request.addfinalizer(cleanup)
```

Comment:
Should we wait for the app to have started, for instance using `wait_for_logs`, as we do for other test containers fixtures like `redis_container`?

Reply:
Currently it's waiting for the health checks defined in the docker-compose file to be healthy. 
in the future we might need some additional checks to ensure Infrahub is completely up and running but for now I think this is good